pyanaconda/modules/payload/dnf/packages/packages_interface.py

```diff
@@ -0,0 +1,194 @@
+        self.implementation.timeout_changed.connect(self.changed("Timeout"))
+        self.implementation.retries_changed.connect(self.changed("Retries"))
+
+    @property
+    def NoCoreGroup(self) -> Bool:
+        """Should the core package group be installed?"""
+        return self.implementation.no_core_group
+
+    @emits_properties_changed
+    def SetNoCoreGroup(self, no_core_group: Bool):
+        """Set if the core package group should be installed."""
+        self.implementation.set_no_core_group(no_core_group)
+
+    @property
+    def DefaultPackageSet(self) -> Bool:
+        """Should the default package set be installed?"""
+        return self.implementation.default_package_set
+
+    @emits_properties_changed
+    def SetDefaultPackageSet(self, default_package_set: Bool):
+        """Set if the default package set should be installed."""
+        self.implementation.set_default_package_set(default_package_set)
+
+    @property
+    def Environment(self) -> Str:
+        """Get environment used for installation."""
+        return self.implementation.environment
+
+    @emits_properties_changed
+    def SetEnvironment(self, environment: Str):
+        """Set environment used for installation."""
+        self.implementation.set_environment(environment)
+
+    @property
+    def GroupList(self) -> List[Str]:
+        """Get list of groups marked for installation."""
+        return self.implementation.group_list
+
+    @emits_properties_changed
+    def SetGroupList(self, group_list: List[Str]):
+        """Set list of groups which will be used for installation."""
+        self.implementation.set_group_list(group_list)
+
+    @property
+    def PackageList(self) -> List[Str]:
+        """Get list of packages marked for installation."""
+        return self.implementation.package_list
+
+    @emits_properties_changed
+    def SetPackageList(self, package_list: List[Str]):
+        """Set list of packages which will be used for installation."""
+        self.implementation.set_package_list(package_list)
+
+    @property
+    def ExcludeGroupList(self) -> List[Str]:
+        """Get list of excluded groups from the installation."""
+        return self.implementation.exclude_group_list
+
+    @emits_properties_changed
+    def SetExcludeGroupList(self, exclude_group_list: List[Str]):
+        """Set list of the excluded groups for the installation."""
+        self.implementation.set_exclude_group_list(exclude_group_list)
+
+    @property
+    def ExcludePackageList(self) -> List[Str]:
+        """Get list of packages excluded from the installation."""
+        return self.implementation.exclude_package_list
+
+    @emits_properties_changed
+    def SetExcludePackageList(self, exclude_package_list: List[Str]):
+        """Set list of packages which will be excluded from the installation."""
+        self.implementation.set_exclude_package_list(exclude_package_list)
+
+    @property
+    def ExcludeDocs(self) -> Bool:
```

Comment:
You don't have a test for `ExcludeDocs`, please check the coverage.

Reply:
Fixed.